src/main/java/Task.java

```diff
@@ -0,0 +1,18 @@
+public class Task {
+    protected String description;
+    protected boolean isDone;
+
```

Comment:
Consider `id` field for Tasks, such that toString() can include `id` automatically